admin/services/data-access/check-window.data.service.js

```diff
@@ -230,6 +230,34 @@ const checkWindowDataService = {
       ))
     }
     return Promise.all(inserts)
+  },
+  /**
+   * Find active check windows
+   * @param checkWindowId
+   * @return {Object}
+   */
+  sqlFindActiveCheckWindows: async (checkWindowId) => {
```

Comment:
Name is plural suggesting an array but return val is a single object.  Suggest calling it `sqlFindOneActiveCheckWindow` instead, or going for plural and returning the array.

Reply:
@jon-shipley updated